src/Plaid/Link/OnSuccessMetadata.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Newtonsoft.Json;
+
+namespace Going.Plaid.Link
+{
+	/// <summary>
+	/// Provides a data structure to use to deserialize the Plaid JSON metadata returned when the onSuccess event
+	/// in the Plaid Link flow is triggered. 
+	/// </summary>
+	public class OnSuccessMetadata
+	{
+
+		/// <summary>
+		/// The public_token is passed to the onSuccess Javascript callback as a separate parameter from 
+		/// the metadata structure itself. It must be manually added to the metadata in order to be 
+		/// included when deserializing.<br/>
+		/// <example>
+		/// <code>
+		///  onSuccess: async function(public_token, metadata)
+		///  {
+		///	   // The metadata object contains info about the institution the
+		///	   // user selected and the account ID or IDs, if the
+		///	   // Select Account view is enabled.
+		///	   // https://plaid.com/docs/#onsuccess-callback
+		///
+		///	   metadata.public_token = public_token;
+		///	  
+		///	   ... send the metadata as JSON to be deserialized into this class ...  
+		///	  
+		///	  }
+		/// </code>
+		/// </example>
+		/// </summary>
+		/// <value>The public token.</value>
+		[JsonProperty("public_token")]
+		public string PublicToken { get; set; } = null!;
+
+		/// <summary>
+		/// Gets or sets the link session identifier.
+		/// </summary>
+		/// <value>The link session identifier.</value>
+		[JsonProperty("link_session_id")]
+		public string LinkSessionId { get; set; } = null!;
+
+		/// <summary>
+		/// Gets or sets the link session identifier.
+		/// </summary>
+		/// <value>The link session identifier.</value>
+		[JsonProperty("account_id")]
+		public string AccountId { get; set; } = null!;
+
+		/// <summary>
+		/// Gets or sets the accounts info.
+		/// </summary>
+		/// <value>The accounts.</value>
+		[JsonProperty("accounts")]
+		[System.Diagnostics.CodeAnalysis.SuppressMessage(
```

Comment:
Remove attribute, and switch to `IReadOnlyList<AccountInfo>`. We'll do another PR shortly to migrate other properties to `IReadOnlyList<>` as well.